traffic_ops/app/lib/API/User.pm

```diff
@@ -174,7 +179,9 @@ sub update {
 		registration_sent 		=> ( $params->{registrationSent} ) ? 1 : 0,
 		role 					=> $params->{role},
 		state_or_province 		=> $params->{stateOrProvince},
-		username 				=> $params->{username}
+		username 				=> $params->{username},
+		tenant_id 				=> $tenant_id
```

Comment:
"There is no limitation currently forcing a single root tenant."

actually, there is, right? i mean, you can't create a tenant thru the api with parent=null. can you? i didn't think you could. i mean, technically, someone can do an insert on the tenant table directly to create multiple "root" tenants but they shouldn't be able to via the tenant api.